movies-app/src/components/Counter/Counter.test.js

```diff
@@ -0,0 +1,86 @@
+import { render, screen } from '@testing-library/react';
+import user from '@testing-library/user-event';
+import Counter from './Counter';
+
+describe('Counter', () => {
+    
+    test('should render correctly', () => {
+        const initialValue = 0;
+        render(<Counter initialCounterValue={initialValue} />);
```

Comment:
Hi @akshaynr ,
Please define the render fn outside the describe block so that it can be reused

Reply:
@shubhamagarwal  Created separate function outside describe block